README.md

```diff
@@ -71,11 +71,11 @@ class Diff extends PureComponent {
 
 | Prop                      | Type                      | Default                        | Description                                                                                                                                                                                                                                                                                                                                                                                                      |
 |---------------------------|---------------------------|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| oldValue                  | `string                   | Object`                        | `''`                                                                                                                                                                                                                                                                                                                                                                                                             | Old value as string (or Object if using `diffJson`).                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
The old readme was not showing up properly because `|` was treated as a table column